RubiconExtension/Generate.swift

```diff
@@ -84,15 +86,38 @@ class Generate: NSObject, XCSourceEditorCommand {
             }
         }
 
-        let text = lines.reduce("", { $0 + "\n" + $1 })
+        let text = lines.joined()
```

Comment:
Proč už zde není potřeba newline?

Reply:
To bylo blbě. Každý řádek na konci obsahuje `\n`. Pro generování mocků to bylo jedno kolik je mezi řádky prázdných řádku, ale pro `tearDown` to musí být dobře.